docs/submit/osg-flock.md

```diff
@@ -1,97 +1,170 @@
-Configuring a submit host to flock to OSG
-=========================================
+Configure an HTCondor submit host to send excess jobs to OSG
+============================================================
+
+This document explains how to add a path for user jobs to flow from your local site out to the OSG,
+which in most cases means that the jobs will have far more resources available to run on than locally.
+If your local batch system frequently has many jobs waiting to run for a long time,
+adding a path to OSG may result in less waiting for your users.
+
+Background
+----------
+Every batch computing system has one or more entry points that users log on to and use to hand over their computing work
+to the batch system for completion.
+For the HTCondor batch system, we say that users log on to a submit host to submit their jobs to HTCondor,
+where they wait ("are queued") until computing resources are available to run them.
+In a purely local HTCondor system, there are one to a few submit hosts and many computing resources.
+
+An HTCondor submit host can also be configured to forward jobs to an OSG-managed submit host.
+This process is called [flocking](https://htcondor.readthedocs.io/en/stable/grid-computing/connecting-pools-with-flocking.html).
+If you already have an HTCondor pool, we recommend that you install this software
+on top of one of your existing HTCondor submit hosts.
+This will let your users submit to the local pool and have the jobs
+automatically get forwarded to OSG when the local pool is full.
+If you do not have an HTCondor batch system, installing this software will give you a submit host
```

Comment:
If …, following these instructions will install the HTCondor submit service and configure it only to forward jobs to OSG. In other words, you do not need a whole HTCondor batch system just to have a local OSG submit host.